extensions/adobe/experience/customerJourneyManagement/message-experienceevent.schema.json

```diff
@@ -0,0 +1,110 @@
+                }
+              },
+              "required": ["https://ns.adobe.com/xdm/experience/customerJourneyManagement/emailChannelContext"]
+            },
+            {
+              "properties": {
+                "https://ns.adobe.com/xdm/experience/customerJourneyManagement/pushChannelContext": {
+                  "title": "Push Channel Context",
+                  "$ref": "https://ns.adobe.com/experience/customerJourneyManagement/push-channel-context",
+                  "description": "Push channel fields for the MessageProfile associated with the Adobe CJM Experience Event."
+                }
+              },
+              "required": ["https://ns.adobe.com/xdm/experience/customerJourneyManagement/pushChannelContext"]
+            }
+          ]
+        },
+        "https://ns.adobe.com/xdm/experience/customerJourneyManagement/eventStatus": {
+          "title": "Event Status",
+          "description": "Provide the status for the Delivery, Feedback, Tracking Experience Event for the MessageProfile.",
+          "oneOf": [
+            {
+              "properties": {
+                "https://ns.adobe.com/xdm/experience/customerJourneyManagement/messageDeliveryfeedback": {
+                  "title": "Message Delivery Feedback",
+                  "$ref": "https://ns.adobe.com/experience/customerJourneyManagement/message-delivery-feedback",
+                  "description": "Captures Delivey status of a message."
+                }
+              },
+              "required": ["https://ns.adobe.com/xdm/experience/customerJourneyManagement/messageDeliveryfeedback"]
+            },
+            {
+              "properties": {
+                "https://ns.adobe.com/xdm/experience/customerJourneyManagement/messageInteraction": {
+                  "title": "Message Interaction",
+                  "$ref": "https://ns.adobe.com/experience/customerJourneyManagement/message-interaction",
+                  "description": "Captures interaction made by the profile on the message."
+                }
+              },
+              "required": ["https://ns.adobe.com/xdm/experience/customerJourneyManagement/messageInteraction"]
+            }
+          ]
+        }
+      }
+    }
+  },
+  "required": [
+    "https://ns.adobe.com/xdm/experience/customerJourneyManagement/messageExecution",
+    "https://ns.adobe.com/xdm/experience/customerJourneyManagement/messageProfile",
+    "https://ns.adobe.com/xdm/experience/customerJourneyManagement/channelContext",
+    "https://ns.adobe.com/xdm/experience/customerJourneyManagement/eventStatus",
+    "xdm:directMarketing"
+  ],
+  "allOf": [
+    {
+      "$ref": "https://ns.adobe.com/xdm/common/extensible#/definitions/@context"
+    },
+    {
+      "$ref": "https://ns.adobe.com/xdm/context/experienceevent"
+    },
+    {
+      "$ref": "https://ns.adobe.com/xdm/context/experienceevent-environment-details"
```

Comment:
It appears you are including other mixins with this current definition. This is okay, but wanted to make sure that is what you wanted.

Reply:
@cmathis : We plan to have "message-experienceevent.schema" as a schema rather a mixin. Is it possible to define and commit pre built schemas like "message-experienceevent.schema".

Also similarly we have used mixin within mixin at line: https://github.com/adobe/xdm/blob/56573ad19fa756030ade3494552c88f3c01543a6/extensions/adobe/experience/customerJourneyManagement/messageprofile.schema.json#L25. I assume, we support adding mixin within mixin.

Reply:
You can have a mixin which reference other mixins so this is fine.

Reply:
@cmathis : Also can you confirm first part of the query:

>  We plan to have "message-experienceevent.schema" as a schema rather a mixin. Is it possible to define and commit pre built schemas like "message-experienceevent.schema".

Reply:
Yes, this is technically possible, but please schedule a meeting with XDM PM and myself to go over the details first.